Update message size limit section (#40138)
mdn/content: files/en-us/web/api/webrtc_api/using_data_channels/index.md

## Understanding message size limits

> It is recommended to keep message sizes moderately small. While most modern browsers support sending messages of at least 256 kilobytes, there are downsides to sending large messages, especially when message interleaving is not available. Without message interleaving (as defined in [RFC 8260](https://www.rfc-editor.org/rfc/rfc8260.html)), sending a large message on one data channel can cause head-of-line blocking, which in turn can negatively affect the latency of messages on other data channels.

Reviewer: ```suggestion
You should keep message sizes moderately small. While most modern browsers support sending messages of at least 256 kilobytes, there are downsides to sending large messages, especially when message interleaving is not available. Without message interleaving (as defined in {{rfc("8260")}}, sending a large message on one data channel can cause {{glossary("head-of-line blocking")}}, which in turn can negatively affect the latency of messages on other data channels.
```
